src/app/EventManagement.h

```diff
@@ -419,6 +423,16 @@ class EventManagement
      */
     CHIP_ERROR EnsureSpaceInCircularBuffer(size_t aRequiredSpace);
 
+    /**
+     * @brief Iterate the event elements inside event tlv and mark the fabric index as 0 if matching with obsolete fabric index
+     * converted from apContext
```

Comment:
"converted" in what sense?

Reply:
removed, update apContext with apFabricIndex, thanks,